lib/bot-builder.js

```diff
@@ -68,6 +69,9 @@ module.exports = function botBuilder(messageHandler, options, optionalLogError)
   if (isEnabled('alexa')) {
     alexaSetup(api, messageHandlerPromise, logError);
   }
+  if (isEnabled('google-hangouts-chat')) {
```

Comment:
I think we should use just `hangouts` because it's shorter but still describes the platform uniquely.